### PR TITLE
Add .travis.yml to test builds on Travis

### PR DESCRIPTION
**Pull request by siteshwar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+services:
+    - docker
+
+before_install:
+    - docker pull ${OS_TYPE}
+
+script:
+    - docker run -v $TRAVIS_BUILD_DIR:/source ${OS_TYPE} bash -c "cd /source;
+        dnf repolist;
+        dnf install -y procps-ng libX11-devel fuse-devel gdbm-devel;
```
Comment (kdudka) [inline .travis.yml]: Why does it install ```libX11-devel``` and ```fuse-devel```?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        dnf groupinstall -y 'C Development Tools and Libraries';
+        XTRAFLAGS='-Wno-unknown-pragmas -Wno-missing-braces -Wno-unused-result -Wno-return-type -Wno-int-to-pointer-cast -Wno-parentheses -Wno-unused -Wno-unused-but-set-variable -Wno-cpp -P';
+        export CCFLAGS=\"-fno-strict-aliasing $XTRAFLAGS\";
+        export CC=gcc;
+        ./bin/package;
+        ./bin/package make mamake || :;
+        ./bin/package make -S 2>&1 | tee out.log ;
+        grep -q 'error:' out.log && exit 1 || exit 0;"
```
Comment (kdudka) [inline .travis.yml]: If I open the build log in Travis CI, it says me:
```
This log is too long to be displayed. Please reduce the verbosity of your build or download the raw log. 
```
Could you please reduce the verbosity somehow?

File: .travis.yml (hunk 1 above)
Comment (siteshwar) [inline .travis.yml]: These are requirements for building See https://github.com/siteshwar/ast/blob/betabeta/src/lib/libtk/unix/tkUnixPort.h#L71 and https://github.com/siteshwar/ast/blob/betabeta/src/cmd/fus3d/fus3d.h#L29

File: .travis.yml (hunk 2 above)
Comment (siteshwar) [inline .travis.yml]: There is ```quiet``` option for package script. However even if the build fails, package script returns 0, so I have to search for errors in build log. So it's not possible to use ```quiet``` option here.

**Pull request merged**